pkg/ctl/create/cluster.go

```diff
@@ -114,8 +119,24 @@ func doCreateCluster(cfg *api.ClusterConfig, name string) error {
 		return fmt.Errorf("--ssh-public-key must be non-empty string")
 	}
 
-	if err := ctl.SetAvailabilityZones(availabilityZones); err != nil {
-		return err
+	if kopsClusterNameForVPC != "" {
+		if len(availabilityZones) != 0 {
+			return fmt.Errorf("--vpc-from-kops-cluster and --zones cannot be used at the same time")
+		}
+		kw, err := kops.NewWrapper(cfg, kopsClusterNameForVPC)
+		if err != nil {
+			return err
+		}
+
+		if err := kw.UseVPC(); err != nil {
```

Comment:
It's not clear that this is altering the cluster config.

I think it would be more clear if we returned the altered version of the ClusterConfig and set it directly to the `ctl.Spec` field.

Reply:
I agree it's not explicit enough. It just seemed very convenient to use `cfg` once and create a pointer to it inside the wrapper struct (less code). See my next comment.